src/Locale/hi/index.js

```diff
@@ -0,0 +1,45 @@
+import Asset from "./Asset.json";
```

Comment:
Do we need this for all languages 🤔 
Can't we write a dynamic import? ie all files in a folder 🤔

Reply:
Updated. Using dynamic imports as much as possible now.

Reply:
Update: Dynamic imports were causing issues during build due to their async nature. Reverted back.

Reply:
I really don't want us to have this file for each language, not a priority to fix now but I think we should find a better solution for this.

Reply:
We can take this up in a later issue. Tracking it through https://github.com/ohcnetwork/care_fe/issues/8705